secrets/encryptedlocal/prompt.go

```diff
@@ -15,10 +15,11 @@ import (
 
 var (
 	ErrInvalidPassword = errors.New(
-		"Password must contain at least one number, one uppercase letter, one special character, and be at least 8 characters long",
+		"password must contain at least one number, one uppercase letter, one special character," +
+			" and be at least 8 characters long",
 	)
-	ErrPasswordMismatch    = errors.New("Passwords do not match")
-	ErrTerminatedOperation = errors.New("Operation terminated")
+	ErrPasswordMismatch    = errors.New("passwords do not match")
+	ErrTerminatedOperation = errors.New("operation terminated")
```

Comment:
We might want to start the error messages with capital letter.

Reply:
Your linters configuration says the opposite.
It's a Go convention: the first character should not be an upper case letter and the last character should not be a punctuation.

Reply:
Yeah, I agree that the linter says the opposite. Thanks for marking this out, I will discuss it with the team and will decide if we will follow this convention.